src/nubank/workspaces/shadow_cljs/target.clj

```diff
@@ -33,7 +25,7 @@
         config
 
         dynamically-resolved-namespaces
-        (find-namespaces-by-regexp state ns-regexp)]
+        (tu/find-test-namespaces state{:ns-regexp ns-regexp})]
```

Comment:
I'd suggest passing the `config` here. Just override the `:ns-regexp` default beforehand if needed. This also supports the `:namespaces` and `:exclude` options which can be specified in the build config.

Reply:
You are right. I fixed it.